e2e_tests/tests/model_hub/test_transformers.py

```diff
@@ -39,3 +39,233 @@ def test_token_classification_ner_amp() -> None:
     config = set_docker_image(config)
 
     exp.run_basic_test_with_temp_config(config, example_path, 1)
+
+
+@pytest.mark.model_hub  # type: ignore
+@pytest.mark.distributed  # type: ignore
+def test_language_modeling_clm() -> None:
+    example_path = conf.model_hub_examples_path("huggingface/language-modeling")
+    config = conf.load_config(os.path.join(example_path, "clm_config.yaml"))
+    config = conf.set_max_length(config, {"batches": 200})
+    config = set_docker_image(config)
+
+    exp.run_basic_test_with_temp_config(config, example_path, 1)
+
+
+@pytest.mark.model_hub  # type: ignore
+@pytest.mark.distributed  # type: ignore
+def test_language_modeling_clm_amp() -> None:
+    example_path = conf.model_hub_examples_path("huggingface/language-modeling")
+    config = conf.load_config(os.path.join(example_path, "clm_config.yaml"))
+    config = conf.set_max_length(config, {"batches": 200})
+    config = conf.set_hparam(config, "use_apex_amp", True)
+    config = set_docker_image(config)
+
+    exp.run_basic_test_with_temp_config(config, example_path, 1)
+
+
+@pytest.mark.model_hub  # type: ignore
+@pytest.mark.distributed  # type: ignore
+def test_language_modeling_mlm() -> None:
```

Comment:
FYI this is marked distributed, but I think the config has `slots_per_trial: 1`. Not sure if that's a typo or not. Made a comment on the config itself.